docs/examples.md

```diff
@@ -263,6 +263,17 @@ build_package_test_task:
   build_package_test_script: python3 setup.py sdist bdist_wheel bdist_egg
 ```
 
+### Linting
+
+You can easily set up linting with Cirrus CI and flake8, here is an example:
+
+```yaml
+lint_task:
+  container:
+    image: jumbocakeyumyum/flake8:latest
```

Comment:
Can you use the official `alpine/flake8:latest` image?

Reply:
oh didn't know that existed, I used my own image 😅

Reply:
Seems legit: https://hub.docker.com/r/alpine/flake8

Reply:
wait no this cant be legit it says `...docker build for fake8 via travis ci`

Reply:
if I remember correctly this isn't travis 😆